### DISCUSSION TITLE
Why Did the Icon Change?

### DISCUSSION THREAD

**Discussion by abbyacknet** [TenderOwl/Frog#191] [Q&A]
The old icon felt right at home on a GNOME desktop and matched the Adwaita styling of the app quite nicely. I see it was updated in commit 78bc72fe64416b807c39ed0da38048a6eb6d589a but cannot find any documented reasoning. When comparing the old and the new, the old icon's use of colors better aligns with the GNOME palette, and the shape itself feels more at home on a system with the Adwaita design language. Why the change?

**Comment by amka** [comment]
Hello @abbyacknet! Thanks for your feedback on the old icon!

The main reason is that during a discussion in the GNOME Circle, I was asked to change the icon to be more in line with the GNOME and Adwaita styles.

Additionally, even though that icon is available under the MIT license, I still think it's nice to have your own.

I made several attempts to redesign it, but my design skills failed. In the end, I gave up and trusted the neural networks, correcting the resulting image.

Well, returning the old one is possible and doesn't seem like a bad idea. At least your review is not the only one that tells me this.

**Reply by abbyacknet** [reply]
Understood. Thank you for taking the time to respond.

Hmm…I think I just found myself a little weekend project. Time to learn Inkscape!

**Comment by hunny-new** [comment]
I change my pinned taskbar icon to use one that fits with my modern papirus theme.
![Screenshot from 2025-05-13 07-22-46](https://github.com/user-attachments/assets/6743e972-8c9a-4b73-b980-8246d7f4f40d)

Spiritually, right hand facing images are deemed forward thinking and moving in a forward direction.
